packages/react-composites/src/composites/CallWithChatComposite/TabHeader.tsx

```diff
@@ -56,6 +56,8 @@ export const TabHeader = (props: TabHeaderProps): JSX.Element => {
   return (
     <Stack horizontal grow styles={mobilePaneControlBarStyle}>
       <DefaultButton
+        aria-role={'button'}
+        aria-live={'polite'}
```

Comment:
Why do we need aria live here? (Can you add a comment what content is being live updated that requires this?)